lib/kion/car.go

```diff
@@ -105,3 +107,20 @@ func GetCARSOnAccount(host string, token string, accID uint) ([]CAR, error) {
 
 	return cars, nil
 }
+
+// GetCARByNameAndAccount returns a car that matches a given name for a given account number
+func GetCARByName(host string, token string, carName string) (CAR, error) {
+	allCars, err := GetCARS(host, token)
+	if err != nil {
+		return CAR{}, err
+	}
+
+	// find our match
+	for _, car := range allCars {
+		if car.Name == carName {
+			return car, nil
+		}
+	}
+
+	return CAR{}, fmt.Errorf("unable to find car %v", carName)
```

Comment:
This method says it accepts an account number, but it does not.  I think this means it will always return the first account with the given carName, which is almost never what the intent is.

Reply:
There's an issue with the Kion API in how it structure the returned CAR objects. See `9996` internally. To get things working for now we can't add any filtering by account number and have to shim the returned CAR with the correct account number later on.